docker-compose-services/php8/README.md

```diff
@@ -0,0 +1,13 @@
+# PHP8 (while in alpha/beta/etc)
+
+DDEV-Local will directly support PHP8 as soon as possible, but not likely until the full release in late 2020. At that time this recipe will be obsolete.
+
+In the meantime, you can use the [official docker php image](https://hub.docker.com/_/php) (currently php:8.0.0alpha3-fpm-buster) to serve PHP.  This can be updated as new versions come out.
+
+1. Copy [docker-compose.php8.yaml](docker-compose.php8.yaml) to your project's .ddev folder.
+2. Copy [nginx-site.conf](nginx-site.conf) to your .ddev/nginx_full directory (overwriting the generated file there)
+3. `ddev start`
+
+You may find that your project has some problems with PHP8 or that PHP8 has some problems with your project :)
```

Comment:
Not sure where to inject this, but the container name isn't web with this recipe. It is php. So to ssh into the container (for example) you have to `ddev ssh -s php`

Reply:
There is a container named "web" and another one named "php", definitely need to mention that though.

Reply:
I added a little mention. But both "web" and "php" are there, and the code is mounted into both in the same place. BUT, the php-fpm running on "web" is irrelevant and ignored.